torch/_torch_docs.py

```diff
@@ -2674,6 +2674,9 @@ def merge_dicts(*dicts):
     Since eigenvalues and eigenvectors might be complex, backward pass is supported only
     for :func:`torch.symeig`
 
+    When :attr:`input` is on CUDA, :func:`torch.eig() <torch.eig>` causes
```

Comment:
This is a great note. 

cc @heitorschueroff who's reviewing operations using MAGMA for host-device sync.